package.json

```diff
@@ -9,6 +9,7 @@
     "deploy:gh-pages": "npm run build:gh-pages && gh-pages -d dist",
     "is-es5": "es-check es5 ./dist/*.js",
     "lint": "eslint --ext .js --ext .jsx .",
+    "lint:fix": "eslint --fix --ext .js --ext .jsx .",
```

Comment:
I added this because doing `npm run lint -- --fix` doesn't work properly for some reason and I wanted the option to auto-fix linter errors.